src/tree/AzureAccountTreeItemWithProjects.ts

```diff
@@ -142,12 +144,15 @@ export class AzureAccountTreeItemWithProjects extends AzureAccountTreeItemBase {
     }
 }
 
-async function getCompiledProjectPath(projectPath: string, projectLanguage: ProjectLanguage): Promise<string | undefined> {
+type CompiledProjectInfo = { compiledProjectPath: string; isIsolated: boolean };
+
+async function getCompiledProjectInfo(projectPath: string, projectLanguage: ProjectLanguage): Promise<CompiledProjectInfo | undefined> {
```

Comment:
Is the idea to make this future proof for other languages? Otherwise I feel like it makes more sense to only call this _if_ we're dealing with a C#/F# project rather than checking it in here.

Reply:
We already use this for Java, too

Reply:
Ya gotta uncollapse some lines of code on Github to see it